test-api-generator/citrus-test-api-core/src/main/java/org/citrusframework/openapi/testapi/OpenApiParameterFormatter.java

```diff
@@ -42,24 +43,34 @@
 import static java.net.URLDecoder.decode;
 import static java.util.Collections.emptyList;
 import static java.util.Collections.singletonList;
+import static java.util.Comparator.comparing;
 import static java.util.stream.Collectors.joining;
 import static org.citrusframework.openapi.testapi.ParameterStyle.DEEPOBJECT;
+import static org.citrusframework.openapi.testapi.ParameterStyle.FORM;
+import static org.citrusframework.openapi.testapi.ParameterStyle.LABEL;
+import static org.citrusframework.openapi.testapi.ParameterStyle.MATRIX;
 import static org.citrusframework.openapi.testapi.ParameterStyle.NONE;
+import static org.citrusframework.openapi.testapi.ParameterStyle.PIPEDELIMITED;
+import static org.citrusframework.openapi.testapi.ParameterStyle.SIMPLE;
+import static org.citrusframework.openapi.testapi.ParameterStyle.SPACEDELIMITED;
 import static org.citrusframework.openapi.testapi.ParameterStyle.X_ENCODE_AS_JSON;
 
 class OpenApiParameterFormatter {
 
     private static final ObjectMapper objectMapper = new ObjectMapper();
-
-    private static final FormatParameters DEFAULT_FORMAT_PARAMETERS = new FormatParameters("", ",");
-    private static final FormatParameters PIPE_DELIMITED_FORMAT_PARAMETERS = new FormatParameters(
-        "", "|");
-    private static final FormatParameters SPACE_DELIMITED_FORMAT_PARAMETERS = new FormatParameters(
-        "", " ");
-    private static final FormatParameters DEFAULT_LABEL_FORMAT_PARAMETERS = new FormatParameters(
-        ".", ",");
-    private static final FormatParameters DEFAULT_LABEL_EXPLODED_PARAMETERS = new FormatParameters(
-        ".", ".");
+    public static final String PARAMETER_NAME_TOKEN = "%PARAMETER_NAME_TOKEN%";
+
+    private static final Map<ParameterStyle, StyleEncoder> ENCODER_LOOKUP = Map.of(
+        NONE, new NoneEncoder(),
+        MATRIX, new MatrixEncoder(),
+        LABEL, new LabelEncoder(),
+        SIMPLE, new SimpleEncoder(new FormatParameters(PARAMETER_NAME_TOKEN+"=", ",")),
```

Comment:
this whole file is not properly formatted.

Reply:
not sure. what do you expect here?